cpp/src/decisiontree/decisiontree_impl.cuh

```diff
@@ -7,7 +7,7 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software
+  Unless required by applicable law or agreed to in writing, software
```

Comment:
knitpick: I think this line was edited by mistake?

Reply:
Yes. Sorry. fixed